themes/default/content/blog/pulumi-cli-time-elapsed/index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "Pulumi CLI now displays time elapsed per resource"
+date: 2022-11-02T11:45:35-06:00
+draft: false
+meta_image: meta.png
+meta_desc: See deployment times across different types of cloud provider resources.
+authors:
+    - kyle-dixler
+tags: 
+  - pulumi
+  - features
+---
+
+If you’ve deployed resources to your favorite cloud provider, you have probably found yourself sitting in the console thinking: “I don’t know how long this is going to take.” Then you deploy the resource and think: “When did I even start this?” When using Pulumi, the CLI prints out how long the update took after it ran, but while you’re in the moment, it feels like ages.
+
+## We’re excited to announce a CLI usability enhancement.
```

Comment:
```suggestion
a CLI usability enhancement.
```

Frame it more concretely in the title, so that folks know what they are reading (or skimming) about?